selectrum.el

```diff
@@ -461,9 +461,12 @@ changes, and is subsequently passed to
 (defvar selectrum--current-candidate-index nil
   "Index of currently selected candidate, or nil if no candidates.")
 
-(defvar selectrum--first-index-displayed nil
+(defvar-local selectrum--first-index-displayed nil
```

Comment:
:+1:

Reply:
I will try to continue make variables default to local with changes when I'm tracing them down anyway and can see this will not change things such they break.